improved readability (#1719)
enkidevs/curriculum: git/essentials/remote-repository/introduction-to-remote-repositories.md

You can have multiple remote repositories, with different permissions on each one. Some can be **read-only** while on some you can perform both **read** and **write** operations.

> These remote repositories are useful and sometimes even a requirement when collaborating as part of a team.

Reviewer: More so "mostly" than "sometimes"

> These remote repositories are useful and sometimes even a requirement when collaborating as part of a team.

Reviewer 2: ```suggestion
These remote repositories are useful and, most of the time, represent a requirement when collaborating as part of a team.
```

Reviewer 2: what do you think of this @nem035?

Reviewer: sure, that works
